### ISSUE TITLE
add pilot3 notes from FDA feedback 

### ISSUE THREAD

**Issue by laxamanaj** [RConsortium/submissions-wg#95]
From the R Consortium R Submissions WG meeting on 03Nov2023, some feedback was given to Pilot 3 submission : 

1. Currently, FDA may test on different R environments (i.e. one reviewer using Rv4.2.1 and other using Rv4.2.3). ADRG states to use Rv4.2.3.
2. Pilot 3 package didn't install using `renv::install()` when using Rv4.2.1, but did install when reviewer was using Rv4.2.3.  
3. With the submission of specifications `adam-pilot-3.xlsx`, this wasn't updated in our ADaM programs to match the file name appropriately.  Reviewers were able to update the file name convention appropriately and adsl.r ran fine.
4. When noting to change source data path in the ADRG, do not include paths to change to in the steps, instead keep note open just to mention to reviewer that "they should change the source data path to the location of where the data was downloaded to from the EDR".

Action : to include these notes in the minutes.

**Comment by saghirb** [comment]
As a follow to the discussions in the WG meeting, would be it useful to include a file with checksums (e.g. MD5 sums) for all the files in this package? It could be useful in identifying any differences post downloaded.

**Comment by laxamanaj** [comment]
> As a follow-up to the discussions in the WG meeting, would be it useful to include a file with check sums (e.g. MD5 sums) for all the files in this package? It could be useful in identifying any differences post downloaded.

Thanks, @saghirb .  Are you talking about running this : https://rdrr.io/r/tools/md5sum.html

**Comment by saghirb** [comment]
> > As a follow-up to the discussions in the WG meeting, would be it useful to include a file with check sums (e.g. MD5 sums) for all the files in this package? It could be useful in identifying any differences post downloaded.
> 
> Thanks, @saghirb . Are you talking about running this : https://rdrr.io/r/tools/md5sum.html

@laxamanaj yes I am. I use it in conjunction with `list.files()` to write (via a data.frame)  a `.md5` file that contains hashes and respective filenames. On Unix based OSes you can verify the integrity of files (data) using `md5sum -c filename.md5`. 

If it helps, I am happy to provide example code.

**Comment by laxamanaj** [comment]
Thanks for your patience in my response @saghirb . Sure , would be interested in a sample code you may have to share. Thanks!

**Comment by saghirb** [comment]
Apologies for the delay in my response @laxamanaj. I have created a GitHub repo with a some Demo code for `tools:md5sum` at:

- [https://github.com/saghirb/md5sum_R_Demo](https://github.com/saghirb/md5sum_R_Demo)

Let me know if this helps.
